test/jdk/javax/swing/plaf/basic/BasicPasswordFieldUI/PasswordSelectionWordTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @key headful
+ * @bug 4231444 8354646
```

Comment:
This bugid needs to be added to java/awt/TextField/SetEchoCharWordOpsTest.java also as that is the original test which raises this issue.

Reply:
I am asking to add "this bugid" 8354646 in SetEchoCharWordOpsTest.java...
I see if we double click at beginning and/or at the end it will identify "One" and "Three" separately which is not desirable and it will fail the test checkpoint "if you are able to move the caret word-by-word press Fail" as caret can be placed in between those words before the fix..

Reply:
> Is this what you're asking for

yes

> If we want that test to cover [8354646](https://github.com/openjdk/jdk/commit/8354646387b87f643935d7372061b8a5eb2db7a1), then we can delete PasswordSelectionWordTest from this PR, right?

No, that is manual..This is automated which is more desirable..